frontend/public/components/autocomplete.tsx

```diff
@@ -3,14 +3,13 @@ import * as classNames from 'classnames';
 import { useDocumentListener, getLabelsAsString } from '@console/shared';
 import { KeyEventModes } from '@console/shared/src/hooks';
 import { fuzzyCaseInsensitive } from './factory/table-filters';
-import { K8sResourceCommon } from '../module/k8s';
 import { TextFilter } from './factory';
 
 const MAX_SUGGESTIONS = 5;
 
-const labelParser = (resources: K8sResourceCommon[]): Set<string> => {
-  return resources.reduce((acc: Set<string>, resource: K8sResourceCommon) => {
-    getLabelsAsString(resource).forEach((label) => acc.add(label));
+const labelParser = (resources: any, labelPath: string): Set<string> => {
```

Comment:
Nit: Even if we need to use `any` here, we can at least make it `any[]` to be slightly better